### PR TITLE
Issue 589: edited to fix southwest home button

### LINKED ISSUE #589: After practicing and checking progress report, modules don't appear on home screen
**Describe the bug**
After practicing and checking progress report, modules don't appear on home screen

**To Reproduce**

- chrome
- ap 1 control
- aolney202202090902
- Computer reads questions and feedback, I type answers
- ch 9
- refutational feedback
- practice for a while
- look at progress report
- go to home

**Expected behavior**

Options for practice modules should always be on home screen

**Screenshots**

**Desktop (please complete the following information):**

- ubuntu 20.04
- Headphones
- Chrome Version 97.0.4692.71 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by JRustyHaner**
Fixes #589

### CODE REVIEW

**Review by MegaGeese — CHANGES_REQUESTED** [review]
This doesnt fix the bug

**Comment by JRustyHaner** [comment]
@MegaGeese for some reason the progress report button kills the curClass.

File: mofacts/client/views/login/signInSouthwest.js [hunk 1]
```diff
@@ -118,6 +118,7 @@ setClass = function(curClassID) {
   const allClasses = Session.get('curTeacherClasses');
   const curClass = allClasses.find((aClass) => aClass.sectionid == curClassID);
   Session.set('curClass', curClass);
+  Session.set('curSectionId', curClass.sectionid)
```
Comment (MegaGeese) [inline mofacts/client/views/login/signInSouthwest.js:121]: This session variable should be cleared when the user logs out with the logout index button

**Review by MegaGeese — CHANGES_REQUESTED** [review]

**Review by MegaGeese — CHANGES_REQUESTED** [review]
Putting it here will only cause the curSectionId to be set to undefined when the user navigates home after a trial. 
A more appropriate place would be in the function call from the logout button click event.

**Pull request merged**